website/src/content/docs/docs/migrate-swagger/01-get-started.md

```diff
@@ -9,43 +9,21 @@ We have created a swagger to TypeSpec conversion tool to help take on the bulk o
 
 **_Important!_** Because TypeSpec is more expressive than Swagger and with the help of evolving Azure libraries, this tool should only be used as an aid in the conversion/migration process, not as the sole tool to produce final version of TypeSpec specs without human inspection, correction and optimization.
 
-## Steps of running the tool
+## Steps of migration and comparison
 
-- Ensure [Node.js](https://nodejs.org/en/download/) 18.3 LTS or later is installed.
-- Install [`@azure-tools/typespec-client-generator-cli`](https://www.npmjs.com/package/@azure-tools/typespec-client-generator-cli):
-
-### Update existing swagger files
-
-- Run the tool to sort existing swagger so you can easily compare with TypeSpec generated swagger. Please note this functionality has been added in version 0.10.0. Please update to latest if you don't see this command option.
+### Prerequisite
 
+- Clone [azure-rest-api-specs](https://github.com/Azure/azure-rest-api-specs).
+- Install dependencies:
   ```shell
-  tsp-client sort-swagger [path to existing swagger]
+  npm install # Run at root
   ```
 
-- Please check in the updated swaggers in separate PR prior submitting TypeSpec. This will allow you and reviewers to easily see any changes introduced by the TypeSpec conversion.
-
 ### Generate TypeSpec with converter
 
-- Install dependencies
-
-  - If you are starting from a scratch folder:
-
-    ```shell
-    npm install @azure-tools/typespec-client-generator-cli
-    ```
-
-    **_Important!_** If it's not your first time install `@azure-tools/typespec-client-generator-cli`, update by
-
-    ```shell
-    npm update @autorest/openapi-to-typespec
-    ```
-
-  - If you are starting from [azure-rest-api-specs](https://github.com/Azure/azure-rest-api-specs):
-    ```shell
-    npm install # Run at root
-    ```
-
-- Run the tool from the directory you would like to output your files.
+- Go to your `specification/{service-name}` folder in `azure-rest-api-specs`.
+- Create a directory holding TypeSpec files. See example for [control-plane](https://github.com/Azure/azure-rest-api-specs/tree/main/specification/contosowidgetmanager/Contoso.Management) and [data-plane](https://github.com/Azure/azure-rest-api-specs/tree/main/specification/contosowidgetmanager/Contoso.WidgetManager).
```

Comment:
this [TypeSpec folder structure](https://github.com/Azure/azure-rest-api-specs/blob/main/documentation/typespec-structure-guidelines.md) is a better choice?  And the contoso sample projects are referred in the guide as well.